console-subscriber/README.md

```diff
@@ -89,6 +89,20 @@ runtime][Tokio] is considered *experimental*. In order to use
   Missing this configuration file during compilation will cause tokio-console to not work, and alternating
   between building with and without this configuration file included will cause
   full rebuilds of your project.
+
+* the `tokio` and `runtime` [`tracing` targets] must be enabled at the [`TRACE`
+  level].
+
+  + if you're using the [`console_subscriber::init()`][init] or
+  [`console_subscriber::Builder`][builder] APIs, these targets are enabled
+  automatically.
+
+  + if you are manually configuring the `tracing` subscriber using the
+  [`EnvFilter`] or [`Targets`] filters from [`tracing-subscriber`], add
+  `"tokio=trace,runtime=trace"` to your filter configuration.
+
+  + also, ensure you have not enabled any of the [compile time filter 
+    features][compile_time_filters] in your `Cargo.toml`.
```

Comment:
The rest of the bullet points in this file are capitalized, so let's capitalize these for consistency:
```suggestion
* The `tokio` and `runtime` [`tracing` targets] must be enabled at the [`TRACE`
  level].

  + If you're using the [`console_subscriber::init()`][init] or
  [`console_subscriber::Builder`][builder] APIs, these targets are enabled
  automatically.

  + If you are manually configuring the `tracing` subscriber using the
  [`EnvFilter`] or [`Targets`] filters from [`tracing-subscriber`], add
  `"tokio=trace,runtime=trace"` to your filter configuration.

  + Also, ensure you have not enabled any of the [compile time filter 
    features][compile_time_filters] in your `Cargo.toml`.
```